examples/c/pwm.c

```diff
@@ -53,13 +54,12 @@ main(void)
         return EXIT_FAILURE;
     }
 
-#if 0  // ROScube-I doesn't support PWM period setting.
+    // ROScube-I doesn't support PWM period setting.
```

Comment:
Will this affect ROScube-I PWM?

Reply:
The new ROScube-I doesn't provide PWM ports.